app/main/data_requests.py

```diff
@@ -31,3 +33,18 @@ def post_ingest(file: FileStorage, data: dict = None) -> Response:
     else:
         current_app.logger.error(f"Bad response: {request_url} returned {response.status_code}")
         return abort(500)
+
+
+def calculate_days_remaining(current_date=datetime.date(datetime.now())):
```

Comment:
Tests still pass if I change `datetime.date(datetime.now())` to just `datetime.now()` so I don't think we need the date conversion

Reply:
I get the following error if I pass just datetime.now():
 TypeError: unsupported operand type(s) for -: 'datetime.date' and 'datetime.datetime'
 Did you change the due date value (datetime.strptime(due_date, "%d/%m/%Y").date()) also when you tested this? The reason I converted them to dates was because otherwise the number of days was rounded down (so it was displaying a date one day behind) if I didn't cut off the timestamp using .date()

Reply:
ah yeah the tests don't use the default values so that's why they passed still. 

We can use this which is slightly more readable `datetime.now().date()` instead of the `datetime.date(datetime.now())`

Reply:
I don't think this has been implemented